app/views/storage_locations/_source.html.erb

```diff
@@ -10,7 +10,8 @@
   label: label,
   error: error,
   selected: storage_location_for_source(source.object),
-  include_blank: true,
+  include_blank: false,
+  prompt: "Choose a storage location",
```

Comment:
Felt like this just looked better.

Before:
<img width="985" alt="image" src="https://github.com/user-attachments/assets/e02aec69-a0a1-44ba-817d-515e575d637e">

After:
<img width="995" alt="image" src="https://github.com/user-attachments/assets/1637de4c-96ba-4440-82cd-979e77b12f69">